app-metrics/node_exporter/node_exporter-1.6.1.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit go-module systemd
+
+DESCRIPTION="Prometheus exporter for machine metrics"
+HOMEPAGE="https://github.com/prometheus/node_exporter"
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/prometheus/node_exporter.git"
+else
+	SRC_URI="https://github.com/prometheus/node_exporter/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
MY_PV is undefined now, can use PV instead.

Reply:
OMG how did it go undetected by me? I guess stored cache distfiles are problem for me. Is there any way to temporary disable distfiles cache for one emerge command?

Done btw